code/modules/mob/living/simple_animal/hostile/goose.dm

```diff
@@ -66,17 +66,21 @@
 	. = ..()
 	goosevomit = new
 	goosevomit.Grant(src)
-	RegisterSignal(src, COMSIG_MOVABLE_MOVED, .proc/goosement)
```

Comment:
Why not just unregister in goosement() then re-register in a timer callback after some reasonable time

Reply:
Why register something to the src obj when I can just use the fucking proc